### ISSUE TITLE
Folder selection widget is inaccessible to keyboard users.

### ISSUE THREAD

**Issue by rebeccacremona** [harvard-lil/perma#1695]
Load /manage/create. Press tab until you are in the "paste url" box. Type something. Then press tab until you are on the folder selection widget. Try to use the keyboard to select a folder.

Expected behavior: spacebar opens dropdown, arrows move between choices, space or enter/return makes a selection.

**Comment by jcushman** [comment]
Good catch. That looks like a vanilla Bootstrap dropdown -- maybe this would do it? http://stackoverflow.com/a/19004486

**Comment by rebeccacremona** [comment]
I like to include Paypal's accessibility plugin in Bootstrap projects. It includes lots of helpful fixes. But, it does alter some of the styling, and I'm not sure how that will interact with the existing theme. And I'm not sure how its alert js will interact with the js for our popup alerts.

https://github.com/paypal/bootstrap-accessibility-plugin

I've been thinking of doing a full a11y review of all the pages, the capture process, etc. Maybe addressing this could be a part of it.

**Comment by jcushman** [comment]
Nice -- looks like bootstrap-accessibility-plugin would indeed add the missing `role="menu"`, and no doubt lots of other useful stuff. Seems worth giving it a try!

**Comment by rebeccacremona** [comment]
Fixed: the anchor tags didn't have href attributes.
